frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -53,11 +53,16 @@ common_info.report_bug = {
     text = _("Report a bug"),
     keep_menu_open = true,
     callback = function()
-        local model = Device.model
-        UIManager:show(InfoMessage:new{
-            text = T(_("Please report bugs to \nhttps://github.com/koreader/koreader/issues\n\nVersion:\n%1\n\nDetected device:\n%2"),
-                version, model),
-        })
+        local msg = T(_("Please report bugs to \nhttps://github.com/koreader/koreader/issues\n\nVersion:\n%1\n\nDetected device:"), version)
+        if Device:isAndroid() then
+            UIManager:show(InfoMessage:new{
+                text = T("%1\n%2\n", msg, Device:info()),
```

Comment:
Wouldn't it make more sense to just change the model variable? In this shape there isn't really anything that could be sensibly localized.